src/main/resources/regression/beir-v1.0.0-robust04.flat.yaml

```diff
@@ -53,3 +53,4 @@ models:
         - 0.3746
       R@1000:
         - 0.6345
+ 
```

Comment:
revert this change, i.e., don't unnecessarily change files

Reply:
Done